doc/api/test.md

```diff
@@ -1461,6 +1461,16 @@ object, streaming a series of events representing the execution of the tests.
 
 Emitted when code coverage is enabled and all tests have completed.
 
+### Event: `'test:dequeue'`
+
+* `data` {Object}
+  * `file` {string|undefined} The path of the test file,
+    undefined if test is not ran through a file.
+  * `name` {string} The test name.
+  * `nesting` {number} The nesting level of the test.
```

Comment:
Nit

```suggestion
  * `level` {number} The nesting level of the test.
```

Reply:
`nesting` is already used on all other events on this class